finding_palindrome.py

```diff
@@ -0,0 +1,22 @@
+"""
+Write a program that check whether a number is palindrome or Not.
+Input string contains only numbers.
+Please work only arithmetic operations, loops and if-condition
+"""
+
+number = input()
+len_number = len(number)
+number = int(number)
+
+palindrome = True
+for left in range(len_number - 1, 0, -2):
```

Comment:
it would look like more simple if we just revert number using arithmetic operators